init.lua

```diff
@@ -9,12 +9,19 @@ if node.getpartitiontable().lfs_size > 0 then
 	 f:flush()
 	 f:close()
 	 file.remove("httpserver-compile.lua")
-	 node.flashreload("lfs.img")
+	 if (node.LFS.reload) then
+	    node.LFS.reload("lfs.img")
+	 else
+	    print "using old flashreload fallback."
+	    print " Consider updating to recent nodeMCU firmware!" 
+	    node.flashreload("lfs.img")
+	 end
```

Comment:
I don't think this is worth keeping around. Especially if it's going to be mostly an untested codepath.

Reply:
I guess you're right.
However, the readme should reflect that current (3.0.0 or newer?) nodemcu-firmware is required.
BTW, I'm also testing a stripped down LFS only version here: https://github.com/langausd/nodemcu-httpserver/tree/LFS-framework

That does definitely break compatibility, but really performs nicely (while serving almost 2 counter.html instances per second, it's even able to serve all six cars wihout crashing...)